docs/source/models.rst

```diff
@@ -486,13 +486,13 @@ Model                             Acc@1          Acc@5
 ================================  =============  =============
 MobileNet V2                      71.658         90.150
 MobileNet V3 Large                73.004         90.858
-ShuffleNet V2 x1.0                68.360         87.582
-ShuffleNet V2 x0.5                57.972         79.780
-ResNet 18                         69.494         88.882
-ResNet 50                         75.920         92.814
-ResNext 101 32x8d                 78.986         94.480
-Inception V3                      77.176         93.354
-GoogleNet                         69.826         89.404
+ShuffleNet V2 x1.0                68.240         87.524
+ShuffleNet V2 x0.5                57.810         79.724
+ResNet 18                         69.500         88.958
+ResNet 50                         75.802         92.764
+ResNext 101 32x8d                 79.020         94.468
+Inception V3                      77.206         93.576
+GoogleNet                         69.702         89.388
```

Comment:
Worth noting that we need to estimate these values with batch-size=1 on 1 GPU to avoid variances introduced due to batch padding (see #4559).